src/main/resources/db/migration/v0.34.0.4__add_vocabulary_gene_to_gene_paralogy.sql

```diff
@@ -0,0 +1,18 @@
+
+INSERT INTO vocabulary (id, name, vocabularylabel) VALUES (nextval('vocabulary_seq'), 'Paralogy confidence','paralogy_confidence');
```

Comment:
These vocabularies and terms almost completely overlap with those for orthology.  Instead of duplication we should  either:
a) in the case of vocabularies where all terms are identical for orthology/paralogy - simply rename the vocabulary to "Homology ...";
or
b) in the case of some terms being specific to either orthology/paralogy - also create a "Homology ..." vocabulary but then have orthology- / paralogy-specific vocabulary term sets
Need to be careful if renaming the vocabulary_label field that this also gets updated in the code (theoretically just in the VocabularyConstants file)